### DISCUSSION TITLE
Scala 3 support for jwt-play

### DISCUSSION THREAD

**Discussion by nielsbuwen** [jwt-scala/jwt-scala#541] [Q&A] [answered]
Hi,

the play framework recently released version `2.9.0-RC1` which now supports Scala 3. In our play application we are using `jwt-play` which does not seem to be available for Scala 3. Do you have any plans to support that?

Greetings

**Comment by erwan** [comment, marked as answer]
hi,

I'm waiting for the stable release, as I want to provide support for the latest stable and the build isn't really setup for multiple versions of a lib.

So as soon as a final version of `2.9.0` is released, I'll activate support for Scala 3 for Play Framework.
